app/code/Magento/GiftMessageGraphQl/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "magento/module-gift-message-graph-ql",
+  "description": "N/A",
+  "type": "magento2-module",
+  "require": {
```

Comment:
For module responsible for handling the GraphQl we need at least the GraphQl module itself. So we need to add it to the requirements. Please, check other GraphQl modules as a reference

Reply:
Added `"magento/module-graph-ql": "*"` as dependency